index.html

```diff
@@ -291,6 +291,42 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer>
+      <div class="contact-form">
+          <a id="contact-form"></a>
+          <div class="contact-section-01">
+          <p class="p-contact">I'm always interested in hearing about 
+              new projects, so if you'd like to chat 
+              please get in.</p>
+          
+      <form action="https://formspree.io/f/mlezeewv" method="POST" id="form">
+          <label for="name"></label>
+          <input 
+          type="text" 
+          id="name" 
+          placeholder="Full name"  
+          name="full-name" maxlength="30" required>
+          <label for="email"></label>
+          <input 
+          type="email" 
+          id="email" 
+          placeholder="Email address" 
+          name="email" required>
+          <label for="text"></label>
+          <textarea 
+          name="message" 
+          id="text" 
+          cols="30" rows="10" 
+          placeholder="Enter text here" maxlength="500" required></textarea>
```

Comment:
Great work implementing the 500 maximum character length on the text area 👍 , and I see that you have included a `required` constraint on the text area to make the field mandatory, good job 💪